src/backend/db/database_manager.py

```diff
@@ -104,6 +104,46 @@ def validate_ticket_id(self, session, ticket_id, return_response=False):
             else:
                 raise ValueError(f"Invalid ticket_id: {ticket_id} does not exist.")
 
+    @staticmethod
+    def create_customer_file_status_table():
```

Comment:
Lets make sure this function is called in the init function of database_manager.py
Also, in the init function call another function to make all the values thread INT DEFAULT NULL, as NULL in common_db.customer_file_status